mmv1/products/compute/OrganizationSecurityPolicy.yaml

```diff
@@ -34,6 +33,7 @@ timeouts:
   update_minutes: 20
   delete_minutes: 20
 custom_code:
+  constants: 'templates/terraform/constants/org_security_policy_parent_diff.go.tmpl'
```

Comment:
This file is not needed. Instead, you can add `ignore_read: true` to `parent` field in the same yaml file to resolve the problem that `parent` is not returned in GA API.

Reply:
We don't need to get the parent somewhere else. `parent` has been set to Terraform state already with the value in Terraform config.

With `ignore_read: true` means that `flatten` is removed, not `expander`. Please let me know if my understanding is wrong.